plugins/node/opentelemetry-instrumentation-runtime/src/metrics/event-loop-lag.ts

```diff
@@ -0,0 +1,99 @@
+/*
+ * Copyright The OpenTelemetry Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import { IntervalHistogram } from "perf_hooks";
+import { Meter } from "@opentelemetry/api-metrics";
+
+export const createEventLoopLagMetrics = (
+  meter: Meter,
+  histogram: IntervalHistogram
+) => {
+  meter
+    .createObservableGauge("process.runtime.nodejs.eventloop.lag", {
+      description: "Lag of event loop.",
+      unit: "seconds",
+    })
+    .addCallback(async (observable) => {
+      const startTime = process.hrtime();
```

Comment:
Since the lowest version of Node is 14, `process.hrtime.bigint()` can be used instead.